expcore/Mini_games.lua

```diff
@@ -183,7 +178,7 @@ function Mini_games.stop_game()
         Event.remove_removable_nth_tick(tick, token)
     end
 
-    for i, player in ipairs(game.connected_players) do
+    for i, _ in ipairs(game.connected_players) do
         game.connected_players[i].teleport({-35,55},"nauvis")
     end
```

Comment:
This should be `for _, player in ipairs` and then remove the `game.connected_players[i]`